ts/Core/Axis/OrdinalAxis.ts

```diff
@@ -518,6 +520,9 @@ namespace OrdinalAxis {
 
                     series.processData.apply(fakeSeries);
 
+                    // Apply grouping if needed.
+                    fireEvent(axis, 'postProcessData', { fakeAxis: fakeAxis });
```

Comment:
- Shouldn't `postProcessData` be fired **after** all series have processed the data?
- I think we shouldn't fire the event on the main axis (I know there's no other callback to `postProcessData` yet, but in the future, it may cause problems). I was hoping we could do something like this: `fireEvent(fakeAxis, 'postProcessData')`, but we can't. How about defining what's in the `postProcessData` callback (line 1039+) as a function, then call the method directly with a different context?